src/Cell.js

```diff
@@ -229,12 +233,23 @@ var Cell = React.createClass({
       'is-dragged-over-down' :  this.isDraggedOverDownwards(),
       'was-dragged-over' : this.wasDraggedOver()
     });
-    return joinClasses(className, extraClasses, updateCellClass);
+    return joinClasses(className, extraClasses);
   },
 
   getUpdateCellClass() {
-    if(this.state.isRowChanging && this.props.selectedColumn != null){
-      return this.props.column.getUpdateCellClass ? this.props.column.getUpdateCellClass(this.props.selectedColumn, this.props.column, this.state.isCellValueChanging) : '';
+    return this.props.column.getUpdateCellClass ? this.props.column.getUpdateCellClass(this.props.selectedColumn, this.props.column, this.state.isCellValueChanging) : '';
+  },
+
+  applyUpdateClass() {
+    var updateCellClass = this.getUpdateCellClass();
+    // -> removing the class
+    if(updateCellClass != null) {
+      this.getDOMNode().classList.remove(updateCellClass);
+      // -> triggering reflow /* The actual magic */
+      // without this it wouldn't work. Try uncommenting the line and the transition won't be retriggered.
+      this.getDOMNode().offsetWidth = element.offsetWidth;
```

Comment:
performance hit is negligible here since this is only applied to those cells that are updating.

Still, it would be preferable to calculate this class on render and apply the highlight animation class rather than after componentDidUpdate  

This is what I was trying to do for a demo, but I didnt get it to work quickly so went with this option instead. 

Another related problem with this, is that once the highlight has been applied to an updated cell. If we try to apply the highlight class again, nothing happens. Even if the class is first removed. 

I will create a github issue in case I forget